Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.java

```diff
@@ -218,6 +220,39 @@ public boolean onTouch(View v, MotionEvent event)
                     ButtonState.PRESSED);
           }
           break;
+        case MotionEvent.ACTION_MOVE:
+          // for Guitar & Drum Overlay Enable Slide Movement
+          if (mPreferences.getInt("wiiController", InputOverlay.OVERLAY_WIIMOTE_NUNCHUK) ==
+                  InputOverlay.OVERLAY_WIIMOTE_GUITAR ||
+                  mPreferences.getInt("wiiController", InputOverlay.OVERLAY_WIIMOTE_NUNCHUK) ==
+                          InputOverlay.OVERLAY_WIIMOTE_DRUM)
+          {
+            int pointerCount = event.getPointerCount();
+            for (int i = 0; i < pointerCount; ++i)
+            {
+              pointerIndex = i;
+              if (button.getBounds()
+                      .contains((int) event.getX(pointerIndex), (int) event.getY(pointerIndex)))
+              {
+                button.setPressedState(true);
+                button.setTrackId(event.getPointerId(pointerIndex));
+                pressed = true;
+                NativeLibrary.onGamePadEvent(NativeLibrary.TouchScreenDevice, button.getId(),
+                        ButtonState.PRESSED);
+              }
+              else
+              {
+                if (button.getTrackId() == event.getPointerId(pointerIndex))
```

Comment:
Any particular reason this isn't a combined `else if`?

Reply:
no i think it just handle two condition for each touch pointer